cachedcomplete/main_script.py

```diff
@@ -20,7 +20,11 @@
 FILES_TO_HASH_INFO = INFO_PREFIX + r'HASH:\s*(?P<files>.*)$'
 
 # Path to the main script
-MAIN_FILE_PATH = os.path.abspath(sys.argv[0])
+if sys.argv[0] != '-c':
```

Comment:
But what if other arguments were also passed to `python`?

Reply:
`sys.argv[0]` is still either `-c` or a filename.

Reply:
But what if I
```bash
python -s -c "print(1)"
```

Then what will be `argv[0]`?

Reply:
`'-c'`